### PR TITLE
Italy 50's

### PR DESCRIPTION
**Pull request by Eeillios**
Added 1953 and 1958 elections and events related to them, San Marino Civil War and eventual showoff with America. The Italian Economic miracle decision was turned into an event reliant on the reconstruction decisions and event 8007751 was reworked to be more accurate. Issues persist with a few events, 8020313, 8020318 and 8020319 crash the game. 8007751, 8020314, 8020320, 8020321, 8020322 don't exist.

### CODE REVIEW

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:512]: Please follow the convention of putting the historical option as the first option in all events and if it is not a historical option, the option with the highest ai_chance should be first. In an event with 3 options, the historical option must have at least 80% ai_chance

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:584]: This is wrong. It should be add_country_modifier = { name = administrative_genius duration = -1 }

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:623]: This is wrong. Please see above.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:662]: This is wrong. Please see above.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:706]: This is wrong, please see above.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:786]: Please change capital = 1731 to tag = TRI

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:782]: Please insert a check for is_vassal = no.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:786]: Please insert a check for is_vassal = no.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:790]: Please insert a check for is_vassal = no.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:794]: Please insert a check for is_vassal = no.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:798]: Please insert a check for is_vassal = no.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:843]: Event option above already has an ai_chance of 0.8 so it should be 0.1 here

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:1006]: Option with highest ai_chance should be ordered first in the list of options

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:1043]: Option with highest ai_chance should be ordered first in the list of options

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:1067]: Should be this instead: RUS = { exists = yes government = proletarian_dictatorship is_greater_power = yes }

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:1069]: Should be this instead: HUN = { exists = yes OR = { government = proletarian_dictatorship1 government = proletarian_dictatorship } }

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:1095]: Please remove this event and restore the decision. Flag check here can be used in the decision.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:1164]: Please do not remove ideology support as that causes crashes.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:1305]: This is wrong, please see above

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:1361]: This is wrong, please see above

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:1402]: I suppose this event relates to Genoa so please insert a check for ITA owning Genoa.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:1464]: This is wrong, please see above

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:1467]: No negative ideology as it will cause crashes.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:1489]: Event option with highest chance should be ordered first in the event's list of options.

**Review by settintotrieste — CHANGES_REQUESTED** [review]

File: CWE/decisions/Dirigisme.txt
Comment (Eeillios) [reply CWE/decisions/Dirigisme.txt:28]: I based it on the USSR reformation/survival events as the fate of the economy in this case would be dependent on actions taken by the player

File: CWE/events/italy.txt
Comment (Eeillios) [reply CWE/events/italy.txt:782]: I didn't put it because OTL Mattei ane ENI became close to the king if Jordan regardless of the protectorate, should I maybe use OR = { is_vassal = no is_vassal = yes)?

File: CWE/events/italy.txt
Comment (Eeillios) [reply CWE/events/italy.txt:786]: See above

File: CWE/events/italy.txt
Comment (Eeillios) [reply CWE/events/italy.txt:790]: See above, also in this case Morocco never gets free before the 60's

File: CWE/events/italy.txt
Comment (Eeillios) [reply CWE/events/italy.txt:794]: See above

File: CWE/events/italy.txt
Comment (Eeillios) [reply CWE/events/italy.txt:798]: See above

File: CWE/events/italy.txt
Comment (Eeillios) [reply CWE/events/italy.txt:1069]: I thought the Hungarian Revolution flag already ensured HUN existed

File: CWE/events/italy.txt
Comment (Eeillios) [reply CWE/events/italy.txt:1164]: How do I make it so an ideology looses support then?

File: CWE/events/italy.txt
Comment (settintotrieste) [reply CWE/events/italy.txt:782]: I have a policy against allowing anyone other than the overlord to affect relations with vassals. It would make sense since foreign policy is handled by the overlord so there’s no way Italy can interact directly with a vassalised Jordan to boost relations between them.

File: CWE/events/italy.txt
Comment (settintotrieste) [reply CWE/events/italy.txt:790]: Same policy applies. There’s an event for Morocco to be independent in 1956 so there’s no need for this.

File: CWE/events/italy.txt
Comment (settintotrieste) [reply CWE/events/italy.txt:1069]: Flags would persist even after annexation.

File: CWE/events/italy.txt
Comment (settintotrieste) [reply CWE/events/italy.txt:1164]: Increasing the ideology of one would reduce the ideology of the others. So conservative ideology is going to be reduced even without this.

File: CWE/decisions/Dirigisme.txt
Comment (settintotrieste) [reply CWE/decisions/Dirigisme.txt:28]: Sorry I want it to be triggered by a decision so the player knows which options to choose when the events pop up.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:1790]: The event that triggers this event chain must check that Austria is independent as this event triggers a war with Italy.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:1627]: Must check to make sure the owner is AUS or ROA in case the allied occupation continues longer than expected.

**Review by settintotrieste — CHANGES_REQUESTED** [review]

File: CWE/events/italy.txt
Comment (Eeillios) [inline CWE/events/italy.txt:2136]: I know every historical event needs an ahistorical option, I will very soon add that but through events that will be triggered by this event

Comment (settintotrieste) [reply CWE/events/italy.txt:2136]: I’m not a fan of a visual novel style of creating events (one option event leading to another) so I prefer if a direct option to avoid changing ruling party ideology is made here so as not to completely box in the player into accepting a centrist party.

File: CWE/localisation/countries.csv
Comment (settintotrieste) [inline CWE/localisation/countries.csv:284]: Please give a more specific name rather than just “Zone B” can call it East Trieste or something but please no generic names.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:2111]: Please put a check that big tent is not already the ruling party

File: CWE/events/disasters.txt
Comment (settintotrieste) [inline CWE/events/disasters.txt:1156]: Please create a event pic for this. My policy is that all historical events should use it’s own unique image.

File: CWE/events/disasters.txt
Comment (settintotrieste) [inline CWE/events/disasters.txt:1169]: Please insert a comment for all option text so it’s easier for future modders to know what each event option is supposed to do.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:1507]: Might be good to add a check for war = no

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:1940]: Please insert check to make sure Italy is a parliamentary republic for all historical election events

File: CWE/events/trieste.txt
Comment (settintotrieste) [inline CWE/events/trieste.txt:131]: Is there a catch all clean up event to remove TRB?

**Review by settintotrieste — CHANGES_REQUESTED** [review]

File: CWE/events/italy.txt
Comment (Eeillios) [reply CWE/events/italy.txt:2136]: The Problem is that OTL there was no way another way a majority could be formed, I tried to look into any other possible coalition but as much as I digged I couldn't find anything that wasn't just the same centrist coalition just with less parties

Comment (Eeillios) [reply CWE/events/italy.txt:2136]: So I really wouldn't know what else to put there

File: CWE/events/trieste.txt
Comment (Eeillios) [reply CWE/events/trieste.txt:131]: Will add it soon

File: CWE/events/disasters.txt
Comment (Eeillios) [reply CWE/events/disasters.txt:1156]: Totally forgot my bad

File: CWE/events/italy.txt
Comment (Eeillios) [reply CWE/events/italy.txt:2136]: I might have found a way but would it be a problem if both options lead to a big_tent government and one of those two can later become of another ideology? alternatively if that's an issue I can make the path that can change ideology later not change ideology during the elections event

File: CWE/events/trieste.txt
Comment (settintotrieste) [reply CWE/events/trieste.txt:131]: Thanks!

File: CWE/events/italy.txt
Comment (settintotrieste) [reply CWE/events/italy.txt:2136]: I see, maybe the other option can just be to call another election (on the premise that a coalition was unable to be formed at all for some reason) then add some militancy here and there plus possibly some ideology support for the socialists. (must also add a election = no check if this is done).

Also, I would a like a trigger check for ITA being run by a big_tent gov for this event as I think the reasons why the election was so close was because the centrist gov faced difficulties so the effects would not make sense for any other party which is in power.

File: CWE/events/trieste.txt
Comment (settintotrieste) [reply CWE/events/trieste.txt:171]: Please also modify the Treaty of Osimo event to annex TRE and TRB.

Comment (Eeillios) [reply CWE/events/trieste.txt:171]: It does already, both are inherited and claims removed

File: CWE/events/italy.txt
Comment (Eeillios) [reply CWE/events/italy.txt:2136]: Yeah another elections soon after is what I was thinking of doing. I don't know how to feel about the big_tent check as that would either mean that if you go with the other choice you don't get elections for that year or I should create an alternate history even choice, which I could do but I'd prefer to do after having finished historical path, so the current solution with the no ideology change could be kept as a compromise until I get on alternate history if it's alright with you

File: CWE/events/trieste.txt
Comment (settintotrieste) [reply CWE/events/trieste.txt:171]: Okay that’s good but I think your changes haven’t been pushed to this PR yet

File: CWE/events/italy.txt
Comment (settintotrieste) [reply CWE/events/italy.txt:2136]: It’s fine not to have an election event for that year if an AH path has been taken because of either an earlier event option or because some generic system intervened. The events should be scripted as tightly as possible because it’s going to be annoying to the player if an event pops up out of context, even more so if they have no option to avoid it.

File: CWE/events/trieste.txt
Comment (Eeillios) [reply CWE/events/trieste.txt:171]: I didn't indeed, now I have

File: CWE/localisation/countries.csv
Comment (settintotrieste) [inline CWE/localisation/countries.csv:284]: I think South Triest is better

**Review by settintotrieste — CHANGES_REQUESTED** [review]

**Comment by Eeillios** [comment]
I'm sorry but I don't think the ideology checks work for the elections system, as they don't take in account if in 1948 someone picks communists but stays democratic and same thing in 1953, secondly if ingame elections elect someone else it alters the whole system and lastly I thought the country flags worked already good enough

**Comment by settintotrieste** [comment]
Sorry, it’s a policy of mine. The bigger the impact of the event, the more tightly scripted it has to be. If an event boxes in the player to accept only the ruling parties offered in the event without any choice to use generic systems to select a different ideology would degrade player agency and make the events fire out of context. Also, the butterfly effect has to be accounted for. This usually means that if the player wants to take the AH path then consequence is that all affected historical paths would be blocked out.

File: CWE/localisation/countries.csv
Comment (settintotrieste) [reply CWE/localisation/countries.csv:284]: I've decided to edit this to South Trieste once this PR is merged. I'm not comfortable with giving tags names that are specific to a particular event chain when the tag can be released outside of the chain through normal indy rebels, dismantlements, liberate country CB etc. The name must be contextual to all possible scenarios.

Comment (Eeillios) [reply CWE/localisation/countries.csv:284]: Ph yes sorry, the last chain was quite a brain drain and I totally forgot about it, I'll do it in the next push

**Comment by Eeillios** [comment]
I understand your policy but I think that if the player doesn't want to be boxed he could just take the ahistorical option in the constitution that locks out all elections, and with the althis I have that in mind but I planned on accounting for all butterflies after historical was done.

**Comment by settintotrieste** [comment]
It’s not good enough to block it with the constitution event unfortunately so I’m afraid my decision is final on this. I’d rather have overkill than underkill for trigger conditions.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:2759]: Please try not to keep the start and end dates too close to each other as sometimes the MTTH doesn't work and the event fires one year later. This is usually when the player saves his game in 1972 to continue later.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:486]: Sorry I can't allow this. As this is a historical event, the conditions must reflect the situation at the time the historical event fired which in this case was DC as the incumbent government.

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:3193]: Missing brackets

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt:3212]: Missing brackets

File: CWE/events/italy.txt
Comment (settintotrieste) [inline CWE/events/italy.txt]: Should be social_reform = X in the effects bracket and wage_and_work_hours = X in the trigger bracket

File: CWE/events/italy.txt
Comment (settintotrieste) [reply CWE/events/italy.txt]: Compromise solution is to check for ideology percentages. If the percentages are similar to the historical result then the event can fire.

**Comment by Eeillios** [comment]
First of all thanks for the patience you had dealing with some of those events and fixing them, I think that soon the PR can be merged after I add the remaining events from 1970 to 1972. Second of all I wanted to ask if it was an issue if I merged two states (Emilia and Romagna) and create a new state out of the Campania one which includes the provinces of Reggio Calabria, Catanzaro and Potenza, as states could shift balances so I felt like I needed the authorization first. Third of all for event 8020340, it is supposed to be used both if the legislature lasts until 1973 or if early elections are called in 1972, is there any way to fire it in both cases, as I have seen triggering it doesn't work and I do not know if OR can be used with dates?

**Comment by settintotrieste** [comment]
No problem and thanks for your contributions, they are very good!

Sure, you can modify the states so long as the total number of states Italy has remains the same at the end of the changes. I'm generally okay if the number of states increase but not if it decreases as the effect of the latter is more problematic due to the hardcoded 8 factory per state restriction. Please do check if the state changes would affect any CBs fired using events as any acquire state CB fired there will need to reference the new state capital of the state to avoid auto invalidation.

Okay, I've modified the flag check to get 8020340 event to fire as intended.

**Review by settintotrieste — APPROVED** [review]

**Pull request merged**
